cli/src/ui.rs

```diff
@@ -626,14 +632,14 @@ impl Ui {
             Some(false) => "(yN)",
             None => "(yn)",
         };
-        let default_choice = default.map(|c| if c { "Y" } else { "N" });
+        let default_index = default.map(|c| if c { 0 } else { 1 });
 
-        let choice = self.prompt_choice(
+        let index = self.prompt_choice(
             &format!("{prompt} {default_str}"),
             &["y", "n", "yes", "no", "Yes", "No", "YES", "NO"],
```

Comment:
(No action needed): This code seems confusing to me. Perhaps there should be a map from strings to a generic type (probably an enum) provided by the user. But this seems unrelated to your PR.

Reply:
I don't like it either. Maybe we can add `.prompt_choice_with()` later. It's weird that uppercase `"Y"` isn't accepted, for example.